### PR TITLE
pydevd-pycharm meta.yaml

### PR DESCRIPTION
**Pull request by cshaley**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by cshaley** [comment]
passing and ready for review https://github.com/orgs/conda-forge/teams/help-python

Adding pydevd-pycharm package to conda forge

**Comment by cshaley** [comment]
@conda-forge/help-python ready for review

**Comment by cshaley** [comment]
@ocefpaf @jakirkham Review / merge?

**Comment by jakirkham** [comment]
@conda-forge-admin, please ping team

@cshaley, please use the command above to request reviews from the team here as opposed to individual pings

File: recipes/pydevd-pycharm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  skip: true  # [win]
+  entry_points:
+    - pydevd = pydevd:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  missing_dso_whitelist:
+    - /lib64/libc.so.6
+    - /lib64/libstdc++.so.6
+    - $RPATH/libc.so.6
+    - $RPATH/libstdc++.so.6
```
Comment (carterbox) [inline recipes/pydevd-pycharm/meta.yaml:22]: This should not be required for a python only package. It looks like the "source" tar has [pre-compiled binaries](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=548317&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=7244). This is not allowed. You must compile libraries with our compilers.

**Review by carterbox — CHANGES_REQUESTED** [review]

**Pull request closed without merging**